code/game/atoms_movable.dm

```diff
@@ -660,6 +660,78 @@
 /atom/movable/proc/Moved(atom/old_loc, movement_dir, forced, list/old_locs, momentum_change = TRUE)
 	SHOULD_CALL_PARENT(TRUE)
 
+	if(prob(0.001))
```

Comment:
please refer to the style documentation
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-early-returns

Reply:
no

Reply:
he makes a good point, this code is too indented